less/admin/AdminHeader.less

```diff
@@ -0,0 +1,19 @@
+.AdminHeader {
+  background: @control-bg;
+  margin-bottom: 20px;
+  padding: 20px 0;
+
+  h2 {
+    margin-top: 0;
+    margin-bottom: 10px;
+    color: @muted-color;
+  }
+
+  .helpText {
+    margin: 0;
+  }
+
+  i {
```

Comment:
Is this for the icon? If so, I'd prefer some semantic selectors based on the icon class